### PR TITLE
Modulesync 0.21.2

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,16 +1,22 @@
 {
-  "name": "darin/zypprepo",
-  "version": "1.0.2",
-  "source": "git://github.com/deadpoint/puppet-zypprepo.git",
-  "author": "darin",
+  "name": "puppet-zypprepo",
+  "version": "2.0.0",
```
Comment (alexjfisher) [inline metadata.json]: 2.0.0-rc0?

File: lib/puppet/type/zypprepo.rb [hunk 2]
```diff
@@ -171,7 +171,7 @@ def self.store
       inifile.each_file do |file|
         current_mode = File.stat(file).mode & 0o777
         unless current_mode == target_mode
-          Puppet.info "changing mode of #{file} from %03o to %03o" % [current_mode, target_mode]
+          Puppet.info "changing mode of #{file} from %03o to %03o" % [current_mode, target_mode] # rubocop:disable Style/FormatString
```
Comment (alexjfisher) [inline lib/puppet/type/zypprepo.rb]: Planning on fixing this later?

**Comment by tampakrap** [comment]
👍

File: metadata.json (hunk 1 above)
Comment (bastelfreak) [inline metadata.json]: 2.0.0 should be fine since that is the version we're releasing next?

File: lib/puppet/type/zypprepo.rb (hunk 2 above)
Comment (bastelfreak) [inline lib/puppet/type/zypprepo.rb]: I'm not sure if this is a false positive match

File: lib/puppet/type/zypprepo.rb [hunk 3]
```diff
@@ -170,11 +167,11 @@ def self.section(name)
     def self.store
       inifile.store
       unless Puppet[:noop]
-        target_mode = 0644 # FIXME: should be configurable
+        target_mode = 0o644 # FIXME: should be configurable
```
Comment (alexjfisher) [inline lib/puppet/type/zypprepo.rb]: Is this definitely right?

File: tests/init.pp [hunk 4]
```diff
@@ -1 +1 @@
-include zypprepo
+include ::zypprepo
```
Comment (alexjfisher) [inline tests/init.pp]: Don't rename tests to examples.  Instead just delete the directory.  The code is a noop.

File: metadata.json (hunk 1 above)
Comment (alexjfisher) [inline metadata.json]: Normally done in the release PR.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
+# puppet-zypprepo
+
+[![Build Status](https://travis-ci.org/voxpupuli/puppet-zypprepo.svg?branch=master)](https://travis-ci.org/voxpupuli/puppet-zypprepo)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/zabbix.svg)](https://forge.puppetlabs.com/puppet/zabbix)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/zabbix.svg)](https://forge.puppetlabs.com/puppet/zabbix)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/zabbix.svg)](https://forge.puppetlabs.com/puppet/zabbix)
```
Comment (alexjfisher) [inline README.md]: umm... Which project is this? ;)

File: README.md [hunk 6]
```diff
@@ -1,3 +1,13 @@
...
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/zabbix.svg)](https://forge.puppetlabs.com/puppet/zabbix)
+
+## Overview
+
 'zypprepo' - The client-side description of a zypper repository.
 
 Usage:
```
Comment (alexjfisher) [inline README.md]: `Usage` could be a heading I suppose.

**Pull request merged**